ReplaySample/src/main/res/raw/exp_configuration.json

```diff
@@ -0,0 +1,28 @@
+{
+	"customerId":"Ij6P1lfZHchO/co10lQ4BQ==",
+	"siteId": 50092,
+	"customerKey": 4453888,
+	"supportsReinvitation":true,
+
+	"replay": {
```

Comment:
This should be updated to DBA

Reply:
In fact, this should really be set up to use the configurator, but that can be done in this followup ticket: https://kanasoftware.jira.com/browse/VOC-35704